references.bib

```diff
@@ -1,9 +1,119 @@
-@article{shannon, 
-	author={Shannon, Claude E.}, 
-	journal={The Bell System Technical Journal}, 
-	title={A mathematical theory of communication}, 
-	year={1948}, 
-	volume={27}, 
-	number={3}, 
-	pages={379-423}
+@article{reifer2003scaling,
+  title={Scaling agile methods},
+  author={Reifer, Donald J and Maurer, Frank and Erdogmus, Hakan},
+  journal={IEEE software},
+  volume={20},
+  number={4},
+  pages={12--14},
+  year={2003},
+  publisher={IEEE}
 }
+
+@inproceedings{begel2007usage,
+  title={Usage and perceptions of agile software development in an industrial context: An exploratory study},
+  author={Begel, Andrew and Nagappan, Nachiappan},
+  booktitle={Empirical Software Engineering and Measurement, 2007. ESEM 2007. First International Symposium on},
+  pages={255--264},
+  year={2007},
+  organization={IEEE}
+}
+
+@article{turk2014limitations,
+  title={Limitations of agile software processes},
+  author={Turk, Dan and France, Robert and Rumpe, Bernhard},
+  journal={arXiv preprint arXiv:1409.6600},
+  year={2014}
+}
```

Comment:
ArXiv is not a journal. Please reformat to an `@misc` entry, as if you were citing a website.